release/test_build_release.sh

```diff
@@ -39,4 +47,14 @@ function test_build_release_handle_automated_build {
 		"true"
 }
 
+function test_build_release_package_release {
```

Comment:
@davisaints, please rename to `test_build_release_has_supported_compressed_files`